content/en/monitors/guide/non_static_thresholds.md

```diff
@@ -0,0 +1,236 @@
+
+{{< tabs >}}
+{{% tab "UI Configuration" %}}
+{{< img src="monitors/guide/non_static_thresholds/seasonal_threshold.png" alt="Configuration to add week_before timeshift to metric query and set formula a/b" style="width:100%;" >}}
+{{% /tab %}}
+
+{{% tab "JSON Example" %}}
+```
+{
+	"name": "[Seasonal threshold] Amount of connection",
+	"type": "query alert",
+	"query": "sum(last_10m):sum:nginx.requests.total_count{env:prod} by {datacenter} / week_before(sum:nginx.requests.total_count{env:prod} by {datacenter}) <= 0.9",
+	"message": "The amount of connection is lower than yesterday by {{value}} !",
+	"tags": [],
+	"options": {
+		"thresholds": {
+			"critical": 0.9
+		},
+		"notify_audit": false,
+		"require_full_window": false,
+		"notify_no_data": false,
+		"renotify_interval": 0,
+		"include_tags": true,
+		"new_group_delay": 60,
+		"silenced": {}
+	},
+	"priority": null,
+	"restricted_roles": null
+}
+```
+{{% /tab %}}
+{{< /tabs >}}
+
+## Spike threshold
+
+### Context
+
+You are in charge of the cost of log management in your company. You want to be alerted when there is a sudden increase in billing for a specific team; you want to capture a spike in log usage.
+
+{{< img src="monitors/guide/non_static_thresholds/graph_log_usage_spike.png" alt="Log usage graph showing a significant increase or spike in log usage" style="width:100%;" >}}
+
+### Monitor
+
+To count the number of indexed logs in Datadog use the following metric: [`datadog.estimated_usage.logs.ingested_events`][4].
+It is an out-of-the-box metric that can be found in your account after logs are sent to Datadog. For more information on log usage metrics, see the [Log Configuration][5] documentation.
+
+The monitor evaluates the sum over the last 5 minutes compared to the average of the last hour. It is composed of 2 queries:
+- Query A represents the last 5 minutes of the metric you want to be aware of, because of the parameter "Evaluate the … of the query over the last 5 minutes".
+- Query B represents the average value of the same metric over the past hour thanks to the moving rollup, which takes the datapoints of the last 4 hours (to have a large amount of datapoints) and calculates the average.
+
+When the value of Query A is much higher than the value of Query B, it's a sign of a substantial and sudden increase in usage which you can capture with a ratio between A and B.
+
+{{< tabs >}}
+{{% tab "UI Configuration" %}}
+
+{{< img src="monitors/guide/non_static_thresholds/spike_threshold.png" alt="Spike threshold config with two log usage metrics" style="width:100%;" >}}
+
+In the formula, divide **Query A** by **Query B** and multiply the quotient by the moving rollup to account for the difference in evaluation windows between the two queries. In this example, since the moving rollup of Query B is 4 hours (240 minutes), you multiply the formula with 48 (240 minutes / 5 minute evaluation window).	
```

Comment:
```suggestion
In the formula, divide **Query A** by **Query B** and multiply the quotient by the moving rollup to account for the difference in evaluation windows between the two queries. In this example, since the moving rollup of Query B is 4 hours (240 minutes), you multiply the formula by 48 (240 minutes / 5 minute evaluation window).	
```